docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -238,6 +238,28 @@ is performance. In practice, users often search for multiple terms at once. For
 '%big%'` can be evaluated directly using a text index by forming the union of the row id lists for terms "little" and "big".
 :::
 
+### Functions Support {#functions-support}
+
+Conditions in the `WHERE` clause contains calls of the functions that operate with columns. If the column is a part of an index, ClickHouse tries to use this index when performing the functions. 
+ClickHouse supports different subsets of functions for the `text` index and supported functions are as follows:
+
+| Function (operator)                                                                         |
```

Comment:
The table basically duplicates the information in mergetree.md. Could we do this instead?
- link from here to the table in mergetree.md as reference information,
- for each of the functions, 1) check if the behavior is intuitive and expected, 2) if it is not and there is no hope it ever will, then remove it (at least for now), 3) then give usage examples for each of the remaining functions here (basically "Fix up the docs", sub-point "Explain which predicates are supported" [here](https://github.com/ClickHouse/clickhouse-private/issues/28172))?